packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -23,54 +23,71 @@ class TestCommand extends FlutterCommand {
   TestCommand({ bool verboseHelp: false }) {
     requiresPubspecYaml();
     usesPubOption();
-    argParser.addOption('name',
+    argParser.addOption(
+      'name',
       help: 'A regular expression matching substrings of the names of tests to run.',
       valueHelp: 'regexp',
       allowMultiple: true,
       splitCommas: false,
     );
-    argParser.addOption('plain-name',
+    argParser.addOption(
+      'plain-name',
       help: 'A plain-text substring of the names of tests to run.',
       valueHelp: 'substring',
       allowMultiple: true,
       splitCommas: false,
     );
-    argParser.addFlag('start-paused',
-        defaultsTo: false,
-        negatable: false,
-        help: 'Start in a paused mode and wait for a debugger to connect.\n'
-              'You must specify a single test file to run, explicitly.\n'
-              'Instructions for connecting with a debugger and printed to the\n'
-              'console once the test has started.'
+    argParser.addFlag(
+      'start-paused',
+      defaultsTo: false,
+      negatable: false,
+      help: 'Start in a paused mode and wait for a debugger to connect.\n'
+            'You must specify a single test file to run, explicitly.\n'
+            'Instructions for connecting with a debugger and printed to the\n'
+            'console once the test has started.',
     );
-    argParser.addFlag('coverage',
```

Comment:
the only real change is adding the  'track-widget-creation' flag
the other changes are just switching the inconsistent idents to match the flutter style guide.

Reply:
thanks